roles/lib_dyn/library/dyn_record.py

```diff
@@ -187,7 +190,7 @@ def main():
             record_type=dict(required=False, type='str', choices=[
                 'A', 'AAAA', 'CNAME', 'PTR', 'TXT']),
             record_value=dict(required=False, type='str'),
-            record_ttl=dict(required=False, default=0, type='int'),
+            record_ttl=dict(required=False, default=3600, type='int'),
```

Comment:
We want the default to remain 0 here because the API will use the default ttl configured for the zone in this case. What we need to do is to get the zone default when ttl=0 and compare the record ttl to that as well.